plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -43,13 +41,20 @@
 
     private IcebergQueryRunner() {}
 
-    public static DistributedQueryRunner createIcebergQueryRunner()
+    public static DistributedQueryRunner createIcebergQueryRunner(TpchTable<?>... tables)
             throws Exception
     {
         return createIcebergQueryRunner(
-                Map.of(),
-                Map.of(),
-                List.of());
+                ImmutableList.copyOf(tables));
+    }
+
+    public static DistributedQueryRunner createIcebergQueryRunner(Iterable<TpchTable<?>> tables)
```

Comment:
rather than adding additional overrides I suggest you use builder directly

Reply:
Overloads are pre-existing. Iv'e just cleaned up a little bit. 
Also this was discussed in second PR. https://github.com/trinodb/trino/pull/10894#discussion_r807731194
As those are 2 PR's https://github.com/trinodb/trino/pull/10894#discussion_r807731194 sharing same base. One will be merged depending on decision if we need Integration or Smoke test.